src/components/addbookbutton.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function AddBookButton({ onClick }) {
+  return (
+    <button type="button" onClick={onClick}>
+      Add Book
+    </button>
+  );
```

Comment:
[OPTIONAL] - You can consider using type="submit" on the submit button so that when the enter key is pressed, the form will submit, improving the user's experience of your website. 👍